qemu/tests/qemu_guest_agent.py

```diff
@@ -29,6 +28,10 @@
 from virttest.utils_windows import virtio_win
 from provider.win_driver_installer_test import (uninstall_gagent,
                                                 run_installer_with_interaction)
+try:
+    from packaging import version
+except ModuleNotFoundError:
+    process.run("pip3 install packaging", shell=True)
 
```

Comment:
I would suggest using the VT module `utils_version.VersionInterval`, we compare version string in VT with this module. Even if it uses the deprecated `distutils`, let's handling the issue in the whole avocado side, I mean `distutils vs packaging`

Reply:
sure. adjust logic with you suggestion, please help to review again thanks.
RESULTS    : PASS 24 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0